docs/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../tsconfig.json",
-  "include": ["next-env.d.ts", "types", "src", "pages", "data"],
+  "include": ["next-env.d.ts", "types", "src", "pages", "data", "../packages/mui-varnish"],
```

Comment:
I imagine they don't do this with other packages because they execute the build steps for this separately.

If we don't do this, and we introduce an error into the varnish components, what happens?

Reply:
ts error show up without this